### PR TITLE
Updated what's new with changes since Sept 7, 2019

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This pull request updates the [What's New](https://devdocs.magento.com/whats-new.html) page with the documentation changes published since the last update on Sept 7.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,64 @@ description: |
...
+  link: https://github.com/magento/devdocs/pull/5360
+- description: Added the [Log locations topic](https://devdocs.magento.com/guides/v2.3/cloud/project/log-locations.html)
+    to the _Cloud Guide_.
+  versions: 2.x
+  type: New topic
+  date: September 13, 2019
+  link: https://github.com/magento/devdocs/pull/5351
+- description: Updated information about using LESS with Grunt in [Compile LESS using
+    Grunt](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css_debug.html)
+    and [Using Grunt for Magento tasks](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/tools/using_grunt.html)
```
Comment (shrielenee) [inline _data/whats-new.yml]: Period

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,64 @@ description: |
...
+  type: Major update
+  date: September 19, 2019
+  link: https://github.com/magento/devdocs/pull/5330
+- description: Added the [Sortable Widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-sortable.html)
+    topic to the _JavaScript Developer Guide_.
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: September 18, 2019
+  link: https://github.com/magento/devdocs/pull/5358
+- description: Added a "Symbolic links" section to the [Contribution Guide](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (meker12) [reply _data/whats-new.yml]: If you look back through the what's new, we are not consistent. Our style guide doesn't provide any guidance about formatting of topic and book titles.

File: _data/whats-new.yml [hunk 3]
```diff
@@ -4,8 +4,64 @@ description: |
...
+  date: September 20, 2019
+  link: https://github.com/magento/devdocs/pull/5438
+- description: Added options and methods to the [Multiselect Widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget-multiselect.html)
+    topic in the _JavaScript Developer Guide_.
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: September 19, 2019
+  link: https://github.com/magento/devdocs/pull/5354
+- description: Updated the [Add site map and search engine robots](https://devdocs.magento.com/guides/v2.3/cloud/trouble/robots-sitemap.html)
+    topic in the _Cloud Guide_ with a VCL snippet example to create a typical domain redirect for `robots.txt`
```
[the remark this answers is not in the document]
Comment (meker12) [reply _data/whats-new.yml]: the sentence continues after `robots.txt`

File: _data/whats-new.yml (hunk 1 above)
Comment (meker12) [reply _data/whats-new.yml]: See line 60.

File: _data/whats-new.yml [hunk 4]
```diff
@@ -4,8 +4,64 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Sep 9 16:43:06 2019
+updated: Tue Sep 24 09:26:19 2019
 entries:
+- description: Added example [introspection queries](https://devdocs.magento.com/guides/v2.3/graphql/queries.html)
```
[the remark this answers is not in the document]
Comment (meker12) [reply _data/whats-new.yml]: I put book title in italics. Our style guide provides no guidance on title formatting.

File: _data/whats-new.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (meker12) [reply _data/whats-new.yml]: Did some investigation on styles-- will add notes to the Style Guide page.

**Comment by meker12** [comment]
running tests

**Pull request merged**